Vagrantfile

```diff
@@ -0,0 +1,43 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+
+
+Vagrant.configure(2) do |config|
+  boxes = {
+    'ubuntu-12.04' => {virtualbox: 'boxcutter/ubuntu1204',  docker: 'ubuntu/precise'},
+    'ubuntu-14.04' => {virtualbox: 'boxcutter/ubuntu1404',  docker: 'ubuntu/trusty'},
+    'centos-6'     => {virtualbox: 'boxcutter/centos67',    docker: 'centos/centos6'},
+    'centos-7'     => {virtualbox: 'boxcutter/centos71',    docker: 'centos/centos7'},
+  }
+
+  boxes.each do |name, box_cfg|
+    config.vm.define name do |vm_config|
+      vm_config.vm.network "private_network", type: "dhcp"
+
+      vm_config.vm.provider :virtualbox do |vb, override|
+        override.vm.box = box_cfg[:virtualbox]
+        override.vm.provision "file", source: "Rakefile", destination: "Rakefile"
+        override.vm.provision "file", source: "passwd", destination: "passwd"
+        override.vm.provision "file", source: "cruise-config.xml", destination: "cruise-config.xml"
+        override.vm.provision "file", source: "pipeline.json", destination: "pipeline.json"
+        override.vm.provision "file", source: "version.json", destination: "version.json"
```

Comment:
There's really no need to copy all the files. I think most of this should be available in the `/vagrant` directory already :)